src/EventManager.php

```diff
@@ -52,7 +54,28 @@ public function dispatchEvent($eventName, ?EventArgs $eventArgs = null)
      */
     public function getListeners($event = null)
     {
-        return $event ? $this->_listeners[$event] : $this->_listeners;
+        if ($event === null) {
+            Deprecation::trigger(
+                'doctrine/event-manager',
+                'TODO',
```

Comment:
https://www.youtube.com/watch?v=CduA0TULnow

Reply:
I totally deserved that. 😓